src/adapters/mc/ssh/Dummy.c

```diff
@@ -0,0 +1 @@
+// Nothing
```

Comment:
Shouldn't check this in

Reply:
He he. No longer necessary but even if it goes in, won't do anything. Removed.